lib/snmplib/mib.c

```diff
@@ -226,10 +226,7 @@ parse_subtree(struct snmp_mib_tree *subtree, char *input, oid *output, int *out_
 }
 
 int
```

Comment:
Let's not drop a potentially useful comment documenting this function parameter:

```suggestion
/// \param out_len number of subid's in "output"
int
```